docs/en/api-guides/usb-serial-jtag-console.rst

```diff
@@ -53,26 +53,26 @@ The USB Serial/JTAG Controller is able to put the {IDF_TARGET_NAME} into downloa
 Limitations
 ===========
 
-There are several limitations to the USB console feature. These may or may not be significant, depending on the type of application being developed, and the development workflow.
+There are several limitations to the USB Serial/JTAG console feature. These may or may not be significant, depending on the type of application being developed, and the development workflow.
 
 {IDF_TARGET_BOOT_PIN:default = "Not Updated!", esp32c3 = "GPIO9", esp32s3 = "GPIO0"}
 
 1. If the application accidentally reconfigures the USB peripheral pins, or disables the USB Serial/JTAG Controller, the device will disappear from the system. After fixing the issue in the application, you will need to manually put the {IDF_TARGET_NAME} into download mode by pulling low {IDF_TARGET_BOOT_PIN} and resetting the chip.
 
-2. If the application enters deep sleep mode, USB CDC device will disappear from the system.
+2. If the application enters deep sleep mode, the USB Serial/JTAG device will disappear from the system.
 
-3. The behavior between an actual USB-to-serial bridge chip and the USB Serial/JTAG Controller is slightly different if the ESP-IDF application does not listen for incoming bytes. An USB-to-serial bridge chip will just send the bytes to a (not listening) chip, while the USB Serial/JTAG Controller will block until the application reads the bytes. This can lead to a non-responsive looking terminal program.
+3. The behavior between an actual USB-to-serial bridge chip and the USB Serial/JTAG Controller is slightly different. A USB-to-serial bridge chip will just send the bytes to a (not listening) chip, while the USB Serial/JTAG Controller will do a one-time wait of up to 50 milliseconds hoping for the terminal to read the bytes. This can add what appears to be a short "pause" in your ESP-IDF application.
```

Comment:
I think the phrase "A USB-to-serial bridge chip will just send the bytes to a (not listening) chip," talks about the data transfer from PC to the ESP. The second part of the sentence now talks about the data transfer from ESP to PC.

I think the original sentence about PC terminal being unresponsive when the ESP doesn't "listen" is still valid here. You can expand the paragraph to explain the delay behavior for the transfer in the other direction.

Reply:
Ah, got it. confusing. maybe I can improve the wording.